spec/parser_spec.py

```diff
@@ -12,6 +12,12 @@
         self.t = Table('table')
         self.p = Parser(self.t)
 
+    with it('parsing a query the origina must be keeped'):
```

Comment:
Typo on "origina".

Reply:
`'parsing a query the original must be kept'`